source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,218 @@
+
+The `default MinIO Operator template <https://github.com/minio/operator/blob/master/kustomization.yaml>`__ provides a starting point for customizing configurations for your local environment.
+You can modify a Kustomize Operator deployment after installation with patches, which are additional YAML configurations applied over the existing base.
+
+
+Prerequisites
+-------------
+
+To install the Operator with Kustomize you will need the following:
+
+* An existing Kubernetes cluster, v1.21 or later.
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Docker <https://docker.com>`__ for your platform.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
+
+This procedure assumes familiarity with the referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.
+
+.. _minio-k8s-deploy-operator-kustomize-repo:
+
+Install the MinIO Operator using Kustomize
+------------------------------------------
+
+The following procedure uses ``kubectl --kustomize`` to install the Operator from the MinIO Operator GitHub repository:
+
+.. important::
+
+   Do not use ``kubectl krew`` or similar methods to update or manage the MinIO Operator installation.
+   If you use Kustomize to install the Operator, you must use Kustomize to manage that installation.
+
+#. Install the latest version of Operator
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      kubectl apply -k github.com/minio/operator\?ref=|operator-version-stable|
+
+   The output resembles the following:
+
+   .. code-block:: shell
+
+      namespace/minio-operator created
+      customresourcedefinition.apiextensions.k8s.io/miniojobs.job.min.io created
+      customresourcedefinition.apiextensions.k8s.io/policybindings.sts.min.io created
+      customresourcedefinition.apiextensions.k8s.io/tenants.minio.min.io created
+      serviceaccount/console-sa created
+      serviceaccount/minio-operator created
+      clusterrole.rbac.authorization.k8s.io/console-sa-role created
+      clusterrole.rbac.authorization.k8s.io/minio-operator-role created
+      clusterrolebinding.rbac.authorization.k8s.io/console-sa-binding created
+      clusterrolebinding.rbac.authorization.k8s.io/minio-operator-binding created
+      configmap/console-env created
+      secret/console-sa-secret created
+      service/console created
+      service/operator created
+      service/sts created
+      deployment.apps/console created
+      deployment.apps/minio-operator created
+
+   Verify the Operator pods are running:
```

Comment:
Should this be a numbered step?
```suggestion
#. Verify the Operator pods are running:
```

Reply:
I hadn't thought of it that way, but a good idea 👍🏻